src/mocks/jest-server.test.js

```diff
@@ -0,0 +1,29 @@
+import { rest } from 'msw';
+import mockConstants from './mock-constants.js';
+import jestServer from './jest-server.js';
+
+const { API_ROOT } = mockConstants;
+
+describe('src/mocks/jest-server', () => {
+  beforeEach(() => {
+    jestServer.use(rest.get(`${API_ROOT}/hello`, (request, response, context) => {
+      return response(
+        context.status(200),
+        context.text('Hello from MSW!'),
+      );
+    }));
+  });
+
+  it('"serves" on http://localhost', async () => {
+    const body = await (await fetch('http://localhost/hello')).text();
+
+    expect(body).toEqual('Hello from MSW!');
+  });
+
+  // This test exists to prove that MSW will fail tests that try to request against an unhandled endpoint. Un-"x"-it if you need to confirm the failure.
```

Comment:
Can it be tested with `expect(() => { ... }).toThrow()`?

Reply:
I've tried a few variations on this now, and it seems like the throw does not happen perhaps at the correct time, or maybe not specifically in the call stack of this function? Either way, it doesn't seem to be capturable in this manner. 😬

Reply:
Nevermind, @bennapp has galaxy brain and fixed it!